libs/coin-modules/coin-aptos/src/__tests__/api/index.integ.test.ts

```diff
@@ -55,4 +55,17 @@ describe("createApi", () => {
       expect(fees.value).toBeGreaterThanOrEqual(0);
     });
   });
+
+  describe("listOperations", () => {
+    it("returns operations from account", async () => {
+      const block = await api.lastBlock();
+
+      const [operations] = await api.listOperations(sender.freshAddress, {
+        minHeight: block.height,
+      });
+
+      expect(Array.isArray(operations)).toBeDefined();
```

Comment:
`Array.isArray` returns a `boolean`, which is not `undefined`. I think this gives a more readable test:
```suggestion
      expect(operations).toBeInstanceOf(Array);
```

Reply:
done